### DISCUSSION TITLE
how to add netbox images

### DISCUSSION THREAD

**Discussion by gabriellramos03** [netbox-community/netbox#15321] [Help Wanted!]
I need help with the netbox to know how to add an image to the netbox and it appears on the rack (I'm learning how to use the platform, I accept courses or video classes) I can even add the image but it doesn't appear on the rack.

![Capturar](https://github.com/netbox-community/netbox/assets/135670319/8d8b9c7f-744e-475d-b7b8-ef92867e8a60)

**Comment by PaulR282** [comment]
You need to add the Image to the device type by editing the device type. There you can add an image for the front and rear.

**Reply by zmaster7** [reply]
To add, you can set the default rack U height/width in pixels in your configuration file, as noted in the documentation:[https://docs.netbox.dev/en/feature/configuration/default-values/](https://docs.netbox.dev/en/feature/configuration/default-values/). You'll want to make sure your images are sized to those dimensions (or multiples of those dimensions) so they don't get distorted.

Here is the documentation for device types that mentions the front/rear images being used for rack elevations: [https://docs.netbox.dev/en/feature/models/dcim/devicetype/](https://docs.netbox.dev/en/feature/models/dcim/devicetype/)
